README.md

```diff
@@ -34,32 +34,73 @@ sh install.sh
 
 _It is recommended to run `install.sh` every time you switch to the `main` branch._
 
-### Start development processes
-
-[dev-process-manager](https://github.com/vivid-planet/dev-process-manager) is used for local development.
+### Build packages
 
-Start Comet Admin packages
+Build all Comet packages
 
 ```bash
-pnpm run dev:admin
+pnpm run build:packages
 ```
 
-Start CMS packages
+### Start development processes
 
-```bash
-pnpm run dev:cms
-```
+[dev-process-manager](https://github.com/vivid-planet/dev-process-manager) is used for local development. We recommend only running the development process you will need.
+Typically, you will need a subset of the available development processes.
 
-It is also possbile to start specific microservices
+Here are a few examples:
 
-```bash
-pnpm run dev:cms:api # (api|admin|site)
-```
+1. You want to add a new component to `@comet/admin`
+
+    Start the development process for `@comet/admin`:
+
+    ```bash
+    npx dev-pm start @comet-admin
+    ```
+
+    Create a development story in Storybook:
+
+    ```bash
+    pnpm run storybook
+    ```
+
+2. You want to add a CMS feature to the API
+
+    Start the development process for `@comet/cms-api`:
+
+    ```bash
+    npx dev-pm start @comet-api
+    ```
+
+    Start Demo API:
+
+    ```bash
+    npx dev-pm start @demo-api
+    ```
+
+    The Demo API will be available at [http://localhost:4000/](http://localhost:4000/)
+
+3. You want to add a CMS feature to the Admin
+
+    Start the development process for `@comet/cms-admin`:
+
+    ```bash
+    npx dev-pm start @comet-admin
+    ```
+
+    Start Demo API and Admin:
+
+    ```bash
+    npx dev-pm start @demo-api @demo-admin
+    ```
+
+    The Demo Admin will be available at [http://localhost:8000/](http://localhost:8000/)
+
+See [dev-pm.config.js](/dev-pm.config.js) for a list of all available processes and process groups.
 
 #### Start Demo
 
 ```bash
-pnpm run dev:demo
+npx dev-pm start @demo
 ```
```

Comment:
If you want to start the demo only, you now have to manually execute the library build. We should add a notice here

Reply:
Same as https://github.com/vivid-planet/comet/pull/1785/files#r1510819375